contrib/check-config.sh

```diff
@@ -83,6 +83,14 @@ check_flags() {
 	done
 }
 
+check_command() {
+	if command -v "$1" >/dev/null 2>&1; then
+		wrap_good "$1 command" 'available'
+	else
+		wrap_bad "$1 command" 'missing'
+	fi
+}
```

Comment:
I really like this! :+1:

I wonder if maybe we should have a `check_device` too (for consistency), something like:
```bash
check_device() {
    if [ -c "$1" ]; then
        wrap_good "$1" 'present'
    else
        wrap_bad "$1" 'missing'
    fi
}
```

(and then using it below for `/dev/zfs`, of course)